src/storage/mod.rs

```diff
@@ -984,6 +1046,9 @@ impl<E: Engine> Storage<E> {
         Ok(())
     }
 
+    /// Do garbage collection, clean up old MVCC keys.
+    /// It's guaranteed that all reads with timestamp > `safe_point` can be performed correctly
+    /// during and after GC.
```

Comment:
```suggestion
    /// during and after the GC operation.
```

Reply:
I believe GC is both a verb and a noun. Gabage Collect / Gabage Collection